AssetManagementTestProject/PageObj/ChangePasswordPage.cs

```diff
@@ -12,21 +12,22 @@ public class ChangePasswordPage : WebDriverAction
     private string btnSaveNewPw = "//button[contains(@type, 'submit')]";
     private string btnChangePw = "//a[contains(@href, '/change-password')]";
     private string btnCancel = "//span[contains(text(), 'Cancel')]";
-    private string headerChangePw = "//h1[text()='Change Password']";
+    public string HeaderChangePw = "//h1[text()='Change Password']";
     private string pathChangePw = "change-password";
-    private string textChangePwSuccessfully = "//p[text()='Your password has been changed successfully']";
+    public string TextChangePwSuccessfully = "//p[text()='Your password has been changed successfully']";
```

Comment:
Make accessbility of all properties in each page object either all public or all private + Follow naming conventions for public/private variables

Reply:
Done